src/core/adapters/SimplePoolAdapter.ts

```diff
@@ -424,7 +424,7 @@ export abstract class SimplePoolAdapter implements IProtocolAdapter {
               const movementValueRaw =
                 (protocolTokenMovementValueRaw *
                   currentTokenPrice.underlyingRateRaw) /
-                BigInt(10 ** currentTokenPrice.decimals)
+                BigInt(10 ** protocolToken.decimals)
```

Comment:
This was wrong before, but we have not noticed due to decimals being the same.

The numbers that we multiply have their own decimals, and we need to divide by both for the final result. Since the movement value if for the underlying, we need to divide by the protocol token. The final step outside the adapter divides by the underlying.